proto/alarm/v1/config.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package alarm.v1;
+
+option go_package = "alarm/config/v1;alarmconfig";
+
+message SendAlarmConfiguration{
+    string config_hash = 1;
+}
+
+message AlarmConfiguration{
+    string config_hash = 1;
+    string type = 2;
+    string on = 3;
+    string info = 4;
+    string chart_id = 5;
+    string units = 6;
+    string exec_path = 7;
+    string recipient = 8;
+    string conf_source = 9;
+    uint64 update_every = 10;
```

Comment:
@MrZammler  should we add this in AlarmLogEntry ?

Reply:
Ok, so the `update_every` (I assume we're talking about the `repeat_every`, i.e. repeat notifications every X).

There is both a netdata.conf configuration entry, that will apply to all alarms, or/and the user can use it in a specific alarm configuration or template. When the item is present in an alarm configuration, it will override what's inside the netdata.conf file.

Furthermore, there are 2 entries, one for repeating warning alerts and one for critical.

From the agent's point of view, it's a configuration entry. But is an alert with this value 10, different from an alert with value 15? (Ship of Theseus!). And should it take part in the hash... ?

If we keep it in configuration, we need 2 of those, one `repeat_every_warn` and one `repeat_every_crit`. I think it would be a hassle for the cloud to check every alarm entry in it's configuration for this key, combine it with the status, and start repeating. For this reason, I think putting it in the AlarmLogEntry keeps things simpler?. But it's fine by me either way.

Reply:
> repeat_every_warn and one repeat_every_crit. I think it would be a hassle for the cloud to check every alarm entry in it's configuration for this key, combine it with the status, and start repeating

I believe we already going to do that, so it would not be a problem. Also I am fine with `repeat_every_warn` and `repeat_every_crit`

> From the agent's point of view, it's a configuration entry. But is an alert with this value 10, different from an alert with value 15? (Ship of Theseus!). And should it take part in the hash... ?

IMHO yes it should be, my rationale regarding this is: if a single new hash is generated when a user modifies an alarm entry, is fine.

Reply:
We will need to think about it... hmmm...

If a user globally (in netdata.conf) defines a repeat of e.g. 10s... then we will send new configurations with this change. So any alerts, will link to that configuration, and you will start repeating notifications messages every 10s.

When the user then disables it, you'll get a new set of configuration files (or, ok, you might already have them). Now, how are we going to stop alerts from step 1 from stop repeating? Unless they change state (and carry the new configuration without repeat), they will still continue to repeat, right?